### PR TITLE
fix: java importing crd fails

### PR DESCRIPTION
**Pull request by campionfellin**
Fixes https://github.com/awslabs/cdk8s/issues/234

Replace `/` with `.`

Tested locally with the CRD example I'm writing for Java.

Signed-off-by: campionfellin <campionfellin@gmail.com>

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/cdk8s-cli/lib/import/base.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export abstract class ImportBase {
 
     for (const name of this.moduleNames) {
       // output the name of the imported resource
-      console.error(name);
+      console.log(name);
```
Comment (eladb) [inline packages/cdk8s-cli/lib/import/base.ts:47]: Is that intentional?

File: packages/cdk8s-cli/lib/import/base.ts [hunk 2]
```diff
@@ -87,9 +87,10 @@ export abstract class ImportBase {
 
           // java!
           if (options.targetLanguage === Language.JAVA) {
+            const javaName = name.replace('/', '.')
```
Comment (eladb) [inline packages/cdk8s-cli/lib/import/base.ts:90]: I never remember if this replaces all occurrences or just the first. I always use a reflex `/\//g`

File: packages/cdk8s-cli/lib/import/base.ts (hunk 1 above)
Comment (campionfellin) [reply packages/cdk8s-cli/lib/import/base.ts:47]: Yeah, I thought the `error` was unintentional. Any reason for `error` over `log` here?

File: packages/cdk8s-cli/lib/import/base.ts (hunk 2 above)
Comment (campionfellin) [reply packages/cdk8s-cli/lib/import/base.ts:90]: Will do that!

File: packages/cdk8s-cli/lib/import/base.ts (hunk 1 above)
Comment (eladb) [reply packages/cdk8s-cli/lib/import/base.ts:47]: It depends whether we want people to be able to capture stdout and do stuff with it. I actually agree that this specific output should go to stdout so I am okay with this change

Comment (campionfellin) [reply packages/cdk8s-cli/lib/import/base.ts:47]: Sounds good 👍

**Comment by campionfellin** [comment]
Will write tests for the base import method tomorrow (or later this week) and update the PR.

**Comment by eladb** [comment]
> Will write tests for the base import method tomorrow (or later this week) and update the PR.

Okay let’s merge this in the meantime and add a test in a subsequent PR

**Pull request merged**